https://issues.redhat.com/browse/ACM-6522 CIM complete restructure (#5605)
open-cluster-management/rhacm-docs: clusters/cluster_lifecycle/cim_intro.adoc

[#cim-intro]
= Creating a host inventory

> You can use the console to create a host inventory, which is a pool of bare metal or virtual machines that you can use to create on-premises {ocp-short} clusters. These clusters can be standalone, with dedicated machines for the control plane, or link:../clusters/hosted_control_planes/hosted_intro.adoc#hosted-control-planes-intro[hosted control planes], where the control plane runs as pods on a hub cluster. You can install standalone clusters by using the console, API, or GitOps by using Zero Touch Provisioning (ZTP). See link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html-single/scalability_and_performance/index#installing-disconnected-rhacm_ztp-preparing-the-hub-cluster[Installing GitOps ZTP in a disconnected environment] in the {ocp} documentation for more information on ZTP.

Reviewer: really long paragraph here with a lot of links. I think we need to break this up.

The host inventory management and on-premises cluster installation are available using the {mce-short} Central Infrastructure Management (CIM) feature. CIM runs the Assisted Installer (also called infrastructure operator) as an operator on the hub cluster.

> 

Reviewer: Is this not a better short description, which would go at the top?

Author: Agreed, I'll switch it around and edit the other paragraph

[#cim-intro]
= Creating a host inventory

> You can use the console to create a host inventory, which is a pool of bare metal virtual machines that you can use to create on-premises {ocp-short} clusters. These clusters can be standalone, with dedicated machines for the control plane, or link:../clusters/hosted_control_planes/hosted_intro.adoc#hosted-control-planes-intro[hosted control planes], where the control plane runs as pods on a hub cluster. You can install standalone clusters by using the console, API, or GitOps by using Zero Touch Provisioning (ZTP). See link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html-single/scalability_and_performance/index#installing-disconnected-rhacm_ztp-preparing-the-hub-cluster[Installing GitOps ZTP in a disconnected environment] in the {ocp} documentation for more information on ZTP.

Reviewer 2: ```suggestion
You can use the console to create a host inventory, which is a pool of bare metal or virtual machines that you can use to create on-premises {ocp-short} clusters. These clusters can be standalone, with dedicated machines for the control plane, or link:../clusters/hosted_control_planes/hosted_intro.adoc#hosted-control-planes-intro[hosted control planes], where the control plane runs as pods on a hub cluster. You can install standalone clusters by using the console, API, or GitOps by using Zero Touch Provisioning (ZTP). See link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html-single/scalability_and_performance/index#installing-disconnected-rhacm_ztp-preparing-the-hub-cluster[Installing GitOps ZTP in a disconnected environment] in the {ocp} documentation for more information on ZTP.
```

> The host inventory management and on-premises cluster installation are available by using the {mce-short} Central Infrastructure Management (CIM) feature. CIM runs the Assisted Installer (also called inftastructure operator) as an operator on the hub cluster.

Reviewer 2: ```suggestion
The host inventory management and on-premises cluster installation are available using the {mce-short} Central Infrastructure Management (CIM) feature. CIM runs the Assisted Installer (also called infrastructure operator) as an operator on the hub cluster.
```

A machine joins the host inventory after booting with a Discovery Image. The Discovery Image is a Red Hat CoreOS live image that containins the following:

- An agent that performs discovery, validation, and installation tasks.
- The necessary configuration for reaching the service on the hub cluster, including the endpoint, token, and static network configuration, if applicable.

You generally have a single Discovery Image for each infrastructure environment, which is a set of hosts sharing a common set of properties. The InfraEnv custom resource definition represents this infrastructure environment and associated Discovery Image.

After the host boots and the agent contacts the service, the service creates a new agent custom resource on the hub cluster representing that host. The agent resources make up the host inventory.

> You can install osts in the inventory as OpenShift nodes later. The agent writes the operating system to the disk, along with the necessary configuration and reboot the host.

Reviewer 2: ```suggestion
You can install hosts in the inventory as OpenShift nodes later. The agent writes the operating system to the disk, along with the necessary configuration, and reboots the host.
```
